src/pudl/package_data/eia861/column_maps/net_metering_eia861.csv

```diff
@@ -4,11 +4,6 @@ utility_id_eia,1,1,1,1,1,1,1,1,1,1,1,1,1,1,2,2,2,2,2,2,2
 utility_name_eia,2,2,2,2,2,2,2,2,2,2,2,2,2,2,3,3,3,3,3,3,3
 short_form,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,4,-1,-1,-1
 state,3,3,3,3,3,3,3,3,3,3,3,3,3,3,1,1,1,1,1,1,1
-residential_total_energy_displaced_mwh,-1,-1,-1,-1,-1,9,9,9,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1
-commercial_total_energy_displaced_mwh,-1,-1,-1,-1,-1,10,10,10,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1
-industrial_total_energy_displaced_mwh,-1,-1,-1,-1,-1,11,11,11,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1
-transportation_total_energy_displaced_mwh,-1,-1,-1,-1,-1,12,12,12,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1
-total_total_energy_displaced_mwh,-1,-1,-1,-1,-1,13,13,13,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1
```

Comment:
You feel pretty confident about this reorganization based on the form documentation?

Reply:
I do - the documentation says: "Net Metering programs allow customers to sell excess power they generate back to the electrical grid to offset consumption" and then lists the "Net Metering Displaced Energy (MWh)" for data entry.

<img width="949" alt="Screen Shot 2024-01-30 at 1 32 10 PM" src="https://github.com/catalyst-cooperative/pudl/assets/49878195/aed30c9a-48e2-469d-86fb-30eba547c36d">

Reply:
And there is no overlap between this "energy_displaced" column and the "sold_to_utiliy" column